### PR TITLE
ESI-11891 Reject gohan startup when there is a migration pending

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

**Comment by mmatczuk** [comment]
Still waiting for vendored code be merged.

**Comment by mmatczuk** [comment]
@przemyslaw-dobrowolski-cl @Zimnx @marcin-ptaszynski please review.

**Comment by marcin-ptaszynski** [comment]
@mmatczuk , this PR makes impossible for users to opt-out migration. Migrations should be opt-in feature, and not mandatory. Developers should be able to run dev server on sqlite3 backend enabled auto-creation of db schema.

**Comment by mmatczuk** [comment]
Vendor code is merged now.

File: server/server.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func (server *Server) initDB() error {
 }
 
 func (server *Server) connectDB() error {
-	if err := migration.Init(); err != nil {
+	if err := migration.EnsureVersion(); err != nil {
```
Comment (przemyslaw-dobrowolski-cl) [inline server/server.go]: it should check no_init flag and reject running only when no_init = true and there are migrations pending

**Comment by mmatczuk** [comment]
Work moved to https://github.com/cloudwan/gohan/pull/429

**Pull request closed without merging**